ambari-server/src/main/python/ambari_server/setupSso.py

```diff
@@ -66,7 +78,7 @@ def validateOptions(options):
     raise FatalException(1, error_msg.format(str(errors)))
 
 
-def populateSsoProviderUrl(options, properties):
+def populate_sso_provide_url(options, properties):
```

Comment:
This should probably be renamed to 'populate_sso_provider_url'

Reply:
yepp; I'll fix it.

Reply:
Fixed